aiosseclient.py

```diff
@@ -20,6 +20,9 @@ async def aiosseclient(url, last_id=None, **kwargs):
                       sock_connect=None, sock_read=None)
     async with aiohttp.ClientSession(timeout=timeout) as session:
         response = await session.get(url, **kwargs)
+        if response.status != 200:
```

Comment:
Maybe something other than 200 also can be OK? Perhaps all the statuses starting 2 and maybe handling 3 also?